spec/datetimeformat.html

```diff
@@ -1250,6 +1254,9 @@ <h1>Properties of Intl.DateTimeFormat Instances</h1>
       <li>[[DateStyle]], [[TimeStyle]] are each either *undefined*, or a String value with values *"full"*, *"long"*, *"medium"*, or *"short"*.</li>
       <li>[[Pattern]] is a String value as described in <emu-xref href="#sec-intl.datetimeformat-internal-slots"></emu-xref>.</li>
       <li>[[RangePatterns]] is a Record as described in <emu-xref href="#sec-intl.datetimeformat-internal-slots"></emu-xref>.</li>
+      <li>[[NumberFormat1]] is a NumberFormat object used for formatting *"numeric"* values.</li>
+      <li>[[NumberFormat2]] is a NumberFormat object used for formatting *"2-digit"* values.</li>
+      <li>[[NumberFormat3]] is a NumberFormat object used for formatting fractional second digits. This slot is only set if *"fractionalSecondDigits"* is set.</li>
```

Comment:
Slight rewording to match the other slot descriptions:
- Prefix with "Intl." match <https://tc39.es/ecma402/#sec-properties-of-intl-relativetimeformat-instances>. (Similar changes should be applied to the other two internal slot descriptions.)
- "It is only used when" to match <https://tc39.es/ecma402/#sec-properties-of-intl-numberformat-instances> and <https://tc39.es/ecma402/#sec-properties-of-intl-datetimeformat-instances>. (DateTimeFormat repeats the slot name [[HourCycle]], but I think it's okay to avoid that repetition.)

```suggestion
      <li>[[NumberFormat3]] is an Intl.NumberFormat object used for formatting fractional second digits. It is only used when [[FractionalSecondDigits]] is not *undefined*.</li>
```